frontend/src/app/[locale]/process/ProcessMilestones.tsx

```diff
@@ -10,17 +10,19 @@ import {
   IconListContent,
   IconListIcon,
   IconListItem,
-  IconListTitle,
 } from "@trussworks/react-uswds";
 
 import ContentLayout from "src/components/ContentLayout";
 import { USWDSIcon } from "src/components/USWDSIcon";
 
 const ProcessMilestones = () => {
-  const t = useTranslations("Process");
+  const t = useTranslations("Process.milestones");
 
-  const messages = useMessages() as unknown as IntlMessages;
-  const keys = Object.keys(messages.Process.milestones.icon_list);
+  const {
+    Process: {
+      milestones: { high_level_roadmap_items },
+    },
+  } = useMessages() as unknown as IntlMessages;
```

Comment:
Might just be a me thing, but I'm not sure I understand what this section is doing, can you explain it a bit?

Reply:
Sure!

As far as I can tell from the docs, `useMessages` will always return all messages in use in the app. Since we only care about the `high_level_roadmap_items` messages, I destructured the return value from the hook to dig into just that part of the object. That way we don't have to use a long chain to index into the object when we want to reference the high_level_roadmap items. Does that make sense? Happy to do a quick call to discuss

Reply:
Thanks for asking and for the explanation. It looks like we could ditch `useMessages()` altogether and just iterate over the three roadmap items.

Reply:
Yeah, this whole thing is kind of odd IMO. My take is we probably shouldn't be directly using the translation messages to determine this functionality. I'd rather have a concept of the roadmap item keys outside of the translation and use those to create translated text.

On the other side of things, `useMessages` seems to be created for the purpose of accessing next intl messages on the client side. Since this is a server component, we likely could access the messages directly as you're describing, but again, I think with a slightly bigger refactor that wouldn't be necessary.

I'll take a crack at it.